system/ee/legacy/libraries/typography/Markdown/Michelf/Markdown.php

```diff
@@ -797,7 +797,7 @@ protected function _doHeaders_callback_setext($matches) {
 		if ($matches[2] == '-' && preg_match('{^-(?: |$)}', $matches[1]))
 			return $matches[0];
 		
-		$level = $matches[2]{0} == '=' ? 1 : 2;
+		$level = $matches[2][0] == '=' ? 1 : 2;
```

Comment:
These need to be converted back to `{}` as well.

Reply:
Fixed on latest commit.

Reply:
@obfuscode Only issue with this is that this is fully deprecated in version 7.4.

RFC: https://wiki.php.net/rfc/deprecate_curly_braces_array_access